### PR TITLE
iOS generateSalt + hashPin

### PR DESCRIPTION
**Pull request by htclifford**

### CODE REVIEW

**Comment by justincrosbiehaventec** [comment]
Looks like there's more included than there needs to be - unrelated Pods (Nimble), built artifacts, etc. Only what is source, and what is required to build the Pod, and the Example app, should be included. Theres too many to itemise, so can you go through and remove anything that is irrelevant?

**Comment by justincrosbiehaventec** [comment]
Can you take a look at the README? Theres refs at the top to Travis etc that need to be removed. See our other repos for examples. 
We need to be consistent in our format across all repos, as much as possible, and within the standard of each specific technology. Please consult the other repos to see where else you need to align for consistency

**Comment by justincrosbiehaventec** [comment]
Example project is failing to compile after following the instructions in the README

File: HaventecCommon/Classes/HaventecCommon.swift [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import Foundation
+import CommonCrypto
+
+public class HaventecCommon {
+    enum HaventecError: Error {
+        case generateSalt
+    }
+    
+    public static func generateSalt(size: Int) throws -> String {
```
Comment (justincrosbiehaventec) [inline HaventecCommon/Classes/HaventecCommon.swift]: In common-android-sdk, the generateSalt method returns a byte[], and is not Base64 encoded (it doesn't actually need to be). Can this do same for consistency?

**Review by justincrosbiehaventec — COMMENTED** [review]
Tests are associated with the Example code, can these be moved to the main Pod code?

File: HaventecCommon/Classes/HaventecCommon.swift (hunk 1 above)
Comment (htclifford) [reply HaventecCommon/Classes/HaventecCommon.swift]: Actioned for the byte[] return

**Pull request merged**